test/new_relic/gemspec_files_test.rb

```diff
@@ -0,0 +1,18 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'minitest/autorun'
```

Comment:
@ajesler - I swapped out the `test_helper` require statement with `minitest/autorun` because we don't want/need the agent to be loaded for this particular test. 

This is what we do with our other utilities, like our [`CveNotifierTests`](https://github.com/newrelic/newrelic-ruby-agent/blob/2fa9a6385f6ee29cda7873890584b5aba6e585f3/test/new_relic/slack_notifications_tests/cve_notifier_tests.rb)